tests/h/views/api_groups_test.py

```diff
@@ -83,8 +91,7 @@ def username_request(self, pyramid_request):
 
     @pytest.fixture
     def group_service(self, pyramid_config):
-        service = mock.Mock(spec_set=['member_leave'])
-        service.member_leave.return_value = None
+        service = mock.create_autospec(GroupService, spec_set=True, instance=True)
```

Comment:
I also changed some pre-existing tests to use `autospec` here (for the leave-a-group view).